src/System.Private.Xml/tests/Xslt/XslTransformApi/CXslTransform.cs

```diff
@@ -1136,40 +1057,32 @@ public void LoadGeneric6(InputType inputType, ReaderType readerType, TransformTy
         [InlineData(InputType.Navigator, ReaderType.XmlValidatingReader, TransformType.Writer, DocType.XPathDocument)]
         [InlineData(InputType.Navigator, ReaderType.XmlValidatingReader, TransformType.Stream, DocType.XPathDocument)]
         [InlineData(InputType.Navigator, ReaderType.XmlValidatingReader, TransformType.TextWriter, DocType.XPathDocument)]
-        [Theory(Skip = "By design bug #84957: Skip this test for Load(url, resolver)")]
-        public void LoadGeneric7(InputType inputType, ReaderType readerType, TransformType transformType, DocType docType)
+        [Theory]
+        public void TC_CustomNullResover_Then_XmlUrlResolver(InputType inputType, ReaderType readerType, TransformType transformType, DocType docType)
         {
-            // By design bug #84957: Skip this test for Load(url, resolver)
-
-            //if (MyInputType() == InputType.URI)
-            //{
-            //    _output.WriteLine("By design bug #84957: Skip this test for Load(url, resolver)");
-            //    return TEST_SKIPPED;
-            //}
-
-            CustomNullResolver myResolver = new CustomNullResolver(null);
-
-            try
+            CustomNullResolver myResolver = new CustomNullResolver(_output);
+            if (inputType == InputType.URI)
             {
-                LoadXSL_Resolver("XmlResolver_Main.xsl", myResolver, inputType, readerType);
+                var e = Assert.Throws<XmlException>(() => LoadXSL_Resolver("XmlResolver_Main.xsl", myResolver, inputType, readerType));
+                var absoluteUri = new Uri(Path.Combine(Environment.CurrentDirectory, FullFilePath("XmlResolver_Main.xsl"))).AbsoluteUri;
```

Comment:
Yeah, this is the cause of the failures on uap, looking at the alternatives here...